.eslintrc

```diff
@@ -77,6 +77,14 @@ rules:
   ## Suggest using 'const' wherever possible
   prefer-const: 2
 
+  # Arrow Functions
+  ## Enforce parens around arrow function arguments
+  arrow-parens: [2, "always"]
+  ## Require a space on each side of arrow operator
+  arrow-spacing: [2, { "before": true, "after": true }]
```

Comment:
and technically the quotes are optional but we don't have consistent use of them in this file...